### PR TITLE
VR-9934 experiment runs read endpoints

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 1]
```diff
@@ -696,4 +696,12 @@ public static ModelDBServiceResourceTypes getModelDBServiceResourceTypesFromRepo
...
     return modelDBServiceResourceTypes;
   }
+
+  public static String getStringFromProtoObjectSilent(MessageOrBuilder messageOrBuilder) {
+    try {
+      return ModelDBUtils.getStringFromProtoObject(messageOrBuilder);
+    } catch (InvalidProtocolBufferException e) {
+      throw new ModelDBException(e);
+    }
+  }
```
Comment (ad-47) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:706]: Comment like other PR

**Review by anandjakhaniya — APPROVED** [review]

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 2]
```diff
@@ -696,4 +696,12 @@ public static ModelDBServiceResourceTypes getModelDBServiceResourceTypesFromRepo
     }
     return modelDBServiceResourceTypes;
   }
+
+  public static String getStringFromProtoObjectSilent(MessageOrBuilder messageOrBuilder) {
+    try {
+      return ModelDBUtils.getStringFromProtoObject(messageOrBuilder);
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:702]:
```suggestion
      return getStringFromProtoObject(messageOrBuilder);
```

**Review by coryverta — APPROVED** [review]

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java [hunk 3]
```diff
@@ -1736,6 +1863,42 @@ public void sortExperimentRuns(
...
+                        SERVICE_NAME,
+                        authService.getVertaIdFromUserInfo(authService.getCurrentLoginUserInfo()),
+                        modelDBServiceActions,
+                        entry.getKey(),
+                        ModelDBServiceResourceTypes.EXPERIMENT_RUN,
+                        Service.MODELDB_SERVICE,
+                        MonitoringInterceptor.METHOD_NAME.get(),
+                        ModelDBUtils.getStringFromProtoObjectSilent(request),
+                        ModelDBUtils.getStringFromProtoObjectSilent(response),
+                        entry.getValue()))
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java]: Anti-pattern.

Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java]: reused other function

**Review by conradoverta — APPROVED** [review]

**Pull request merged**